backend/openapi.yml

```diff
@@ -2437,6 +2605,21 @@ components:
         - id
         - can_id
         - fiscal_year
+    CreateUpdateFundingReceived:
+      description: A request object for the creation or updating of a FundingReceived
+      type: object
+      properties:
+        fiscal_year:
+          type: integer
+        can_id:
+          type: integer
+        budget:
```

Comment:
Either you need to change the budget in this to funding, or in your exampling you need to change funding to budget. Either way, just appears to be a minor issue with inconsistency of naming between your CreateUpdate object and the Example.